API-Reference/Kokkos::create_mirror.md

```diff
@@ -44,30 +57,57 @@ ImplMirrorType create_mirror_view_and_copy(Space const& space, ViewType const&);
   typename ViewType::HostMirror create_mirror(ViewType const& src);
   ```
   Creates a new host accessible `View` with the same layout and padding as `src`.
-  * `src`: a `Kokkos::View`. 
+  * `src`: a `Kokkos::View`.
+
+* ```cpp
+  template <class ViewType>
+  typename ViewType::HostMirror create_mirror(decltype(Kokkos::ViewAllocateWithoutInitializing()),
+                                              ViewType const& src);
+  ```
+  Creates a new host accessible `View` with the same layout and padding as `src`. The new view will have unitialized data.
+  * `src`: a `Kokkos::View`.
 
 * ```cpp
   template <class Space, class ViewType>
   ImplMirrorType create_mirror(Space const& space, ViewType const&);
   ```
   Creates a new `View` with the same layout and padding as `src` but with a device type of `Space::device_type`.
-  * `src`: a `Kokkos::View`. 
+  * `src`: a `Kokkos::View`.
   * `Space`: a class meeting the requirements of `ExecutionSpaceConcept` or `MemorySpaceConcept`
-  * `ImplMirrorType`: an implementation defined specialization of `Kokkos::View`. 
+  * `ImplMirrorType`: an implementation defined specialization of `Kokkos::View`.
+
+* ```cpp
+  template <class Space, class ViewType>
+  ImplMirrorType create_mirror(decltype(Kokkos::ViewAllocateWithoutInitializing()),
+                               Space const& space, ViewType const&);
+  ```
+  Creates a new `View` with the same layout and padding as `src` but with a device type of `Space::device_type`. The new view will have unitialized data.
+  * `src`: a `Kokkos::View`.
+  * `Space`: a class meeting the requirements of `ExecutionSpaceConcept` or `MemorySpaceConcept`
+  * `ImplMirrorType`: an implementation defined specialization of `Kokkos::View`.
 
 * ```cpp
   template <class ViewType>
   typename ViewType::HostMirror create_mirror_view(ViewType const& src);
   ```
   If `src` is not host accessible (i.e. if `SpaceAccessibility<HostSpace,ViewType::memory_space>::accessible` is `false`)
   it creates a new host accessible `View` with the same layout and padding as `src`. Otherwise returns `src`.
-  * `src`: a `Kokkos::View`. 
+  * `src`: a `Kokkos::View`.
+
+* ```cpp
+  template <class ViewType>
+  typename ViewType::HostMirror create_mirror_view(decltype(Kokkos::ViewAllocateWithoutInitializing()),
+                                                   ViewType const& src);
+  ```
+  If `src` is not host accessible (i.e. if `SpaceAccessibility<HostSpace,ViewType::memory_space>::accessible` is `false`)
+  it creates a new host accessible `View` with the same layout and padding as `src`. The new view will have unitialized data. Otherwise returns `src`.
```

Comment:
Typo
```suggestion
  it creates a new host accessible `View` with the same layout and padding as `src`. The new view will have uninitialized data. Otherwise returns `src`.
```